nix/uv-binary/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib,
+  stdenvNoCC,
+  fetchurl,
+}:
+let
+  version = "0.2.17";
```

Comment:
This needs a bump already. We need to figure out some way to make this easier to update given how frequently new versions come out.

Reply:
I refactored the source handling into a common file and added a script that automates the update at `python/update-uv-sources.sh`.

Reply:
That's a great approach, thank you!